src/components/ChannelList/utils.ts

```diff
@@ -6,26 +6,26 @@ import type {
 } from 'stream-chat';
 
 type MoveParameters<
-  At extends UnknownType = UnknownType,
-  Ch extends UnknownType = UnknownType,
-  Co extends string = LiteralStringForUnion,
-  Ev extends UnknownType = UnknownType,
-  Me extends UnknownType = UnknownType,
-  Re extends UnknownType = UnknownType,
-  Us extends UnknownType = UnknownType
+  At extends UnknownType = DefaultAttachmentType,
```

Comment:
were these defaults already being imported here?

Reply:
oh missed that one on the imports, did a global replace all